stack/stack_orion_gnu.yaml

```diff
@@ -0,0 +1,358 @@
+
+cdo:
+  build: NO 
+  version: 1.9.8
+
+pio:
+  build: YES
+  version: 2.5.2
+  enable_pnetcdf: NO
+  enable_gptl: NO
+
+esmf:
+  build: YES
+  version: ESMF_8_2_0
+  install_as: 8_2_0
+  shared: YES
+  enable_pnetcdf: NO
+  debug: NO
+
+fms:
+  build: YES
+  version: 2021.03
+  repo: noaa-gfdl
+  cmake_opts: "-DGFS_PHYS=ON -D64BIT=ON -DOPENMP=ON"
+
+bacio:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+sigio:
+  build: YES
+  version: v2.3.2
+  install_as: 2.3.2
+  is_nceplib: YES
+
+sfcio:
+  build: YES
+  version: v1.4.1
+  install_as: 1.4.1
+  is_nceplib: YES
+
+gfsio:
+  build: YES
+  version: v1.4.1
+  install_as: 1.4.1
+  is_nceplib: YES
+
+w3nco:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+sp:
+  build: YES
+  version: v2.3.3
+  install_as: 2.3.3
+  openmp: ON
+  is_nceplib: YES
+
+ip:
+  build: YES
+  version: v3.3.3
+  install_as: 3.3.3
+  openmp: ON
+  is_nceplib: YES
+
+ip2:
+  build: YES
+  version: v1.1.2
+  install_as: 1.1.2
+  openmp: ON
+  is_nceplib: YES
+
+landsfcutil:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+nemsio:
+  build: YES
+  version: v2.5.2
+  install_as: 2.5.2
+  is_nceplib: YES
+
+nemsiogfs:
+  build: YES
+  version: v2.5.3
+  install_as: 2.5.3
+  is_nceplib: YES
+
+w3emc:
+  build: YES
+  version: v2.7.3
+  install_as: 2.7.3
+  is_nceplib: YES
+
+g2:
+  build: YES
+  version: v3.4.3
+  install_as: 3.4.3
+  is_nceplib: YES
+
+g2c:
+  build: YES
+  version: v1.6.4
+  install_as: 1.6.4
+  is_nceplib: YES
+
+g2tmpl:
+  build: YES
+  version: v1.10.0
+  install_as: 1.10.0
+  is_nceplib: YES
+
+crtm:
+  build: YES
+  version: v2.3.0
+  install_as: 2.3.0
+  is_nceplib: YES
+  install_fix: NO
+
+upp:
+  build: YES
+  version: upp_v10.0.10
+  install_as: 10.0.10
+  openmp: ON
+  is_nceplib: NO
+
+wrf_io:
+  build: YES
+  version: v1.2.0
+  install_as: 1.2.0
+  openmp: ON
+  is_nceplib: YES
+
+bufr:
+  build: YES
+  version: bufr_v11.5.0
+  install_as: 11.5.0
+  python: YES
+  is_nceplib: YES
+
+wgrib2:
+  build: YES
+  version: 2.0.8
+  install_as: 2.0.8
+  lib: YES
+  ipolates: 3
+
+prod_util:
+  build: YES
+  version: v1.2.2
+  install_as: 1.2.2
+  is_nceplib: YES
+
+grib_util:
+  build: YES
+  version: v1.2.3
+  install_as: 1.2.3
+  openmp: ON
+  is_nceplib: YES
+
+ncio:
+  build: YES
+  version: v1.0.0
+  install_as: 1.0.0
+  is_nceplib: YES
+
+boost:
+  build: YES
+  version: 1.68.0
+  level: headers-only
+
+eigen:
+  build: YES
+  version: 3.3.7
+
+gsl_lite:
+  build: YES
+  version: 0.37.0
+
+gptl:
+  build: NO
+  version: 8.0.2
+
+fftw:
+  build: NO
+  version: 3.3.8
+
+tau2:
+  build: NO
+  version: 3.25.1
+
+cgal:
+  build: NO
+  version: 5.0.2
+
+json:
+  build: YES
+  version: 3.9.1
+
+json_schema_validator:
+  build: YES
+  version: 2.1.0
+
+pybind11:
+  build: YES
+  version: 2.5.0
+
+ecbuild:
+  build: YES
+  version: 3.6.1
+  repo: ecmwf
+
+eckit:
+  build: YES
+  version: 1.16.0
+  repo: ecmwf
+
+fckit:
```

Comment:
And if this is just for srw-v2, I think we should disable these JEDI builds that aren't sepcifically needed.

Reply:
Which libraries belong to Jedi?

Reply:
I will update the branch name or will create another one `release/srw-public-v2`